test/gha/README.md

```diff
@@ -0,0 +1,17 @@
+## Enabling Github Action Runs in Your AWS Account
+
+1. Deploy the [Cloudformation stacks](https://github.com/aws/karpenter/tree/main/test/gha/cloudformation/README.md) into your account that enable Managed Prometheus and the Github Actions runner policies.
+2. Remove the `if: github.repository == 'aws/karpenter'` lines from the top of the `e2e-matrix.yaml` file so that the jobs are allowed to run in other repositories besides Karpenter
```

Comment:
Wonder if we could drive this via another mechanism. Maybe the existence of a gh secret?

Reply:
We _could_ just not gate it. This will outright fail if it tries to assume the role outlined in the action. I'm fine either way.

Reply:
I'm going to do a little look-around and see if there's a best practice on how to handle actions like this for forks.